### DISCUSSION TITLE
Wayland Global Hotkey workaround

### DISCUSSION THREAD

**Discussion by JordanPlayz158** [obsproject/obs-studio#7076] [Development Questions] [answered]
Wayland currently doesn't have global hotkeys so while we wait for this feature, perhaps we could add a middle ground which would not only fix this problem but also make scripting even easier. Perhaps we could add a cli or maybe just a command specifically for saving replay buffer (but I think a general cli makes more sense to control the currently opened instance). Replay-Sorcery, the application I moved from did this exact approach, it had `replay-sorcery save` and you would add it to your keyboard shortcuts in your Wayland DE and then it would save no matter which application you are focused in, perhaps there is a way to add a command dynamically through the current scripting system but I think a cli would all around improve the experience.

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
